README.md

```diff
@@ -30,24 +41,25 @@ To setup firebase on our apps we will need to perform the following:
 
 Our React app is going to use a `service` ( just a function ) to create a 
 `firebaseAuth()` instance and call the `signInWithPopUp()` 
-method and pass it an instance of a `GoogleAuthProvider()`.   
+method and pass it an instance of a `GoogleAuthProvider()`. 
```

Comment:
Could you provide a screenshot of what the final app would look like here for students to get a better visual?